packages/modules/order/src/migrations/Migration20240827133639.ts

```diff
@@ -3,7 +3,15 @@ import { Migration } from "@mikro-orm/migrations"
 export class Migration20240827133639 extends Migration {
   async up(): Promise<void> {
     this.addSql(
-      'alter table if exists "return_item" add column if not exists "damaged_quantity" numeric not null default 0, add column if not exists "raw_damaged_quantity" jsonb not null;'
+      'alter table if exists "return_item" add column if not exists "damaged_quantity" numeric not null default 0, add column if not exists "raw_damaged_quantity" jsonb;'
```

Comment:
q: this will also fail if there is existing data in the table right? we need to set default for `raw_damaged_quantity` here